functions/gm.fish

```diff
@@ -0,0 +1,3 @@
+function gm
+  git commit -am "$argv"
```

Comment:
if we keep the " sign you can type on terminal: gm Commit message
if we take out the " sign you can type on terminal gm "Commit message"

do you like more without " sign?